public/app/plugins/datasource/cloudwatch/tracking.ts

```diff
@@ -0,0 +1,92 @@
+import { DashboardLoadedEvent } from '@grafana/data';
+import { reportInteraction } from '@grafana/runtime';
+
+import { isCloudWatchLogsQuery, isCloudWatchMetricsQuery } from './guards';
+import pluginJson from './plugin.json';
+import { CloudWatchMetricsQuery, CloudWatchQuery, MetricEditorMode, MetricQueryType } from './types';
+
+interface CloudWatchOnDashboardLoadedTrackingEvent {
+  grafana_version?: string;
+  dashboard_id?: string;
+  org_id?: number;
+
+  /* The number of CloudWatch logs queries present in the dashboard*/
+  logs_queries_count: number;
+
+  /* The number of CloudWatch metrics queries present in the dashboard*/
+  metrics_queries_count: number;
+
+  /* The number of queries using the "Search" mode. Should be measured in relation to metrics_queries_count */
+  metrics_search_count: number;
```

Comment:
My point here is that the `metrics_search_count` in it self may not be interesting, but if you also look at the `metrics_queries_count` you can derive the proportions. For example, 
```
metrics_queries_count=10;
metrics_search_count=3;
metrics_queries_count=7:
``` 
Out of 10 metric queries, 3 are search queries and 7 are insights queries. We could calculate the percentages here, but I think it will be easier to write sql queries if we save that for later. 

I'm very open to suggestions on how to improve the comments here so that they're easier to understand. :)

Reply:
that makes sense!